pom.xml

```diff
@@ -177,7 +177,51 @@
     </profile>
   </profiles>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>4.12</version>
```

Comment:
sorry, this version upgrade slipped in (was 4.11), though I think it might be ok. I did not update the other versions.